R/package.R

```diff
@@ -2,7 +2,10 @@
 #'
 #' @import hyperSpec
 #' @import hySpc.testthat
-
+#' @importFrom utils head tail
```

Comment:
sort "importFrom" tags alphabetically.

Reply:
On the other hand, shouldn't these `#' @importFrom` tag be uset in the documentation of a function where the imported functions are forst used? I think, we discussed this some time ago during video meeting.

@bryanhanson, do you remember if there was an issue or guideline created for this?

Reply:
Yes, at the time (a few weeks ago), I recall we had decided that it was better to have the imports in the file that requires them.  Duplicates are automatically eliminated.  However, today it seemed that CB was less certain in her thoughts.  My thought is that this should be left until much later, we don't have time to perfect this if we want these packages released.  If it works, don't break it.

Reply:
OK, @sangttruong, just sort "importFrom" tags alphabetically.